dev/com.ibm.ws.request.timing.hung_fat/fat/src/com/ibm/ws/request/timing/hung/fat/HungRequestTiming.java

```diff
@@ -263,18 +257,26 @@ public void testHungDynamicThresholdUpdate() throws Exception {
         server.setServerConfigurationFile("server_hungRequestThreshold3.xml");
         server.waitForStringInLogUsingMark("CWWKG0017I", 90000);
 
-        CommonTasks.writeLogMsg(Level.INFO, "Calling TestWebApp Application with URL=" + url.toString());
-        con = getHttpConnection(url);
-        br = getConnectionStream(con);
-        br.readLine();
+        createRequest(4000);
 
         CommonTasks.writeLogMsg(Level.INFO, "Waiting for hung detection warning");
-        CommonTasks.writeLogMsg(Level.INFO, "----> Hung Request Warning 1 : " + line1);
-        server.waitForStringInLogUsingMark("TRAS0114W", 30000);
+        server.waitForStringInLog("TRAS0114W", 30000);
```

Comment:
Should we set `setMarkToEndOfLog()` before we change server configuration and should this line use `waitForStringInLogUsingMark()`?

Reply:
Updated